readme.md

```diff
@@ -1003,6 +1003,71 @@ echo $faker->vat;           // "A35864370"
 echo $faker->dni; // '83367512'
 ```
 
+### `Faker\Provider\fa_AF\Person`
+
+```php
+<?php
```

Comment:
Please mention only the formatters specific to your locale, so e.g. remove `lastName` as it's already documented in the main `Person` section.

Reply:
I fixed the locale formatters in the readme file. Please let me know if any further modifications are needed.

Reply:
Most of formatters you mentioned in the locale-specific section exist in every locale (see https://github.com/fzaninotto/Faker/blob/master/src/Faker/Generator.php for the list). You should only document those that are specific to your locale : `firstNameAscii`, `lastNameAscii`, and `cityName`.

Also, there are some non-ascii characters in the output of the `firstNameAscii` documentation:

![image](https://user-images.githubusercontent.com/99944/33707698-8c908a92-db38-11e7-89e4-c8994f5339b4.png)